backend/app/tests/services/llm/test_chain_executor.py

```
@@ -0,0 +1,215 @@
+
+    def test_run_failure_sends_callback(self, context, request_obj, failure_result):
+        executor = self._make_executor(context, request_obj, failure_result)
+
+        with (
+            patch("app.services.llm.chain.executor.Session") as mock_session,
+            patch("app.services.llm.chain.executor.send_callback") as mock_callback,
+            patch("app.services.llm.chain.executor.update_llm_chain_status"),
+        ):
+            mock_session.return_value.__enter__.return_value = MagicMock()
+
+            result = executor.run()
```

Ruff (0.15.2)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)